inc/classes/subscriber/Cache/class-expired-cache-purge-subscriber.php

```diff
@@ -67,13 +67,38 @@ public function __construct( Options_Data $options, Expired_Cache_Purge $purge )
 	 */
 	public static function get_subscribed_events() {
 		return [
-			'init'                => 'schedule_event',
-			'rocket_deactivation' => 'unschedule_event',
-			static::EVENT_NAME    => 'purge_expired_files',
-			'cron_schedules'      => 'custom_cron_schedule',
+			'init'                            => 'schedule_event',
+			'rocket_deactivation'             => 'unschedule_event',
+			static::EVENT_NAME                => 'purge_expired_files',
+			'cron_schedules'                  => 'custom_cron_schedule',
+			'update_option_' . WP_ROCKET_SLUG => [ 'clean_expired_cache_scheduled_event', 10, 2 ],
```

Comment:
@crystinutzaa For timing, does it matter when this callback fires in relationship to any 3rd party or other callbacks? 

What do I mean?

Do we need it to run:

- before everything else, ie run first? If yes, then the priority would be `0`.
- after everything else, ie run last? If yes, then the priority would be `PHP_INT_MAX`.

If it doesn't matter (ie there's not a potential of timing), then a priority of `10` is fine.

What do you think?

Reply:
@hellofromtonya this should not have any impact to any 3rd party or other callbacks. Basically it will simply reschedule the deletion of WP Rocket cache to the new lifespan.